RecoTBCalo/EcalTBRecProducers/interface/EcalTBWeightUncalibRecHitProducer.h

```diff
@@ -37,8 +37,8 @@ class EcalTBWeightUncalibRecHitProducer : public edm::EDProducer {
     EcalUncalibRecHitRecWeightsAlgo<EBDataFrame> EBalgo_;
     EcalUncalibRecHitRecWeightsAlgo<EEDataFrame> EEalgo_;
 
-    const EEShape testbeamEEShape;  
-    const EBShape testbeamEBShape; 
+    EEShape testbeamEEShape;  
+    EBShape testbeamEBShape; 
```

Comment:
it looks like this change is not needed here because these shapes are never modified after the construction

Reply:
Yes. I updated this. Now it has the const. It will be fixed in the next updated of the PR.